scss/main.scss

```diff
@@ -1123,10 +1123,16 @@ p.cardText {
   }
 
   h2 {
-    color: #333;
     font-size: 2rem;
     padding-bottom: 16px;
     padding-top: 0px;
+    margin-bottom: 0px;
+    &:hover {
```

Comment:
Two things:
1. Can we move this pseudo-class to the anchor element instead?
2. Can we style the `:focus` pseudo-class on the anchor element as well?

Basically, we should have the underline also appear if a user is navigating with a keyboard instead of a mouse.

Reply:
moved hover and added focus pseudo classes to anchor tag